src/sdam/srv_polling.ts

```diff
@@ -106,15 +103,14 @@ export class SrvPoller extends TypedEventEmitter<SrvPollerEvents> {
     }
   }
 
+  // TODO(NODE-4817): This method has been left to allow for refactoring with the new logger
```

Comment:
TODOs normally refer to tickets that haven't been done yet so we can find the relevant code again. I think this should probably be something in the new spec-ed logger epic